debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/RecordsSnapshotProducer.java

```diff
@@ -148,7 +149,7 @@ private void takeSnapshot(Consumer<SourceRecord> consumer) {
             // we're locking in SHARE UPDATE EXCLUSIVE MODE to avoid concurrent schema changes while we're taking the snapshot
             // this does not prevent writes to the table, but prevents changes to the table's schema....
             schema.tables().forEach(tableId -> statements.append("LOCK TABLE ")
-                                                         .append(tableId.toString())
+                                                         .append(tableId.toQuotedId(PostgresTableIdTransformer.INSTANCE))
```

Comment:
I'm a bit confused by this one, does it mean the table name always gets quoted now? If so, I don't think that's right. But I may well be missing something.